op-batcher/batcher/espresso.go

```diff
@@ -192,6 +194,49 @@ func (s *espressoTransactionSubmitter) SubmitTransaction(job *espressoCommon.Tra
 	}
 }
 
+// Evaluation result for a job.
+type JobEvaluation int
+
+const (
+	// Continue handling the current job.
+	Handle JobEvaluation = iota
+	// Retry the submission.
+	RetrySubmission
```

Comment:
I think we should merge `RetrySubmission` and `RetryVerification` into Retry.

Reply:
These two retries are actually different. In the original code, lines 291 and 295 corresponded to two different cases, where we would either retry the submission or the verification, so I added both types here.

Reply:
I understand now why we need both.